### PR TITLE
Localize All Text

### PR DESCRIPTION
**Pull request by ayubun**
Hello hello!

The PR is rather straightforward, I think ^^ Just an update for all of the content in interns/dev to become localized with the changes made via https://github.com/LokiLabs/microsoft-graph-explorer-teams-app/pull/15

**To test:**
See https://github.com/LokiLabs/microsoft-graph-explorer-teams-app/pull/15 for testing steps!

**Note:**
There is a bug with some of the RSC list not being localized. This is a separate matter, as the bug existed prior to this PR (See [AD#39416](https://dev.azure.com/microsoftgarage/Intern%20GitHub/_backlogs/backlog/GI21%20-%20Graph%20Explorer/Backlog%20items/?workitem=39416))

--

*Ticket:*
[AD#39307](https://dev.azure.com/microsoftgarage/Intern%20GitHub/_backlogs/backlog/GI21%20-%20Graph%20Explorer/Backlog%20items/?workitem=39307)

### CODE REVIEW

File: tabs/src/components/sample/AzureFunctions.jsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ var functionName = process.env.REACT_APP_FUNC_NAME || "myFunc";
 
 async function callFunction() {
   try {
+    // Not fully sure how to localize this since it is not a react component
```
Comment (ayubun) [inline tabs/src/components/sample/AzureFunctions.jsx]:
```suggestion
```

File: tabs/src/components/sample/ConnectedResources.jsx [hunk 2]
```diff
@@ -4,9 +4,12 @@ import { ClipboardCopiedToIcon } from '@fluentui/react-icons-northstar'
 import copy from "copy-to-clipboard"; 
 import * as microsoftTeams from "@microsoft/teams-js"; 
 import "./ConnectedResources.css";
-import {TEAMS_CHANNEL_ID, CHAT, CHAT_ID,  NO_CONNECTED_RESOURCES} from './TabConstants';
+import { useTranslation } from "react-i18next";
 
 export function ProcessTeamsContext(){
+
+    // Translations
+    const { t } = useTranslation();
```
Comment (acchiang) [inline tabs/src/components/sample/ConnectedResources.jsx:12]: Thoughts on renaming this `t` variable to something more descriptive?

File: tabs/src/translations/zh-CN.json [hunk 3]
```diff
@@ -1,15 +1,70 @@
 {
     "translation": {
+
+        "Query Runner": {
+            "Key": "(zh-cn) Key",
```
Comment (acchiang) [inline tabs/src/translations/zh-CN.json:5]: mmm this is definitely chinese hahahah

File: tabs/src/components/sample/ConnectedResources.jsx (hunk 2 above)
Comment (ayubun) [reply tabs/src/components/sample/ConnectedResources.jsx:12]: This is what react-i18next docs use, so I think its standard practice.

File: tabs/src/translations/zh-CN.json (hunk 3 above)
Comment (lhao03) [reply tabs/src/translations/zh-CN.json:5]: It's Chinese that I can finally read 😎

**Review by lhao03 — APPROVED** [review]
lgtm! 
` \^o^/`

**Review by acchiang — APPROVED** [review]
thanks for clarifying @AyuAi. LGTM

**Review by jerryxihe — APPROVED** [review]

**Review by csyan5 — APPROVED** [review]

File: tabs/src/translations/en-US.json [hunk 4]
```diff
@@ -1,6 +1,58 @@
...
+        "Query Runner": {
+            "Key": "Key",
+            "Value": "Value",
+            "Run query": "RUN QUERY",
+            "Request body": "Request Body",
+            "Request headers": "Request Headers",
+            "Response body": "Response Body",
+            "Response headers": "Response Headers",
+            "Response": "Response",
+            "Warning": "To use Graph requests, you must connect the app to a resource in order to use resource-specific consent queries. Learn more here.",
```
Comment (Lauragra) [inline tabs/src/translations/en-US.json]: Revise the text of the warning: "You must connect the app to a resource in order to make Microsoft Graph requests that use resource-specific consent. For details, see {link to name of article}." We avoid using "here" as link text due to accessibility.
```suggestion
            "Warning": "You must connect the app to a resource in order to make Microsoft Graph requests that use resource-specific consent. For details, see {link to name of article(placeholder)} .",
```

File: tabs/src/translations/en-US.json [hunk 5]
```diff
@@ -1,6 +1,58 @@
...
+            "Add": "ADD"
+        },
+
+        "Connected Resources": {
+            "Teams Channel ID": "Teams Channel ID",
+            "Chat": "Chat",
+            "Chat ID": "Chat ID",
+            "No Connected Resources": "No connected resources. Please connect to a Teams or Chats instance."
```
Comment (Lauragra) [inline tabs/src/translations/en-US.json]: In this message are you referring to Teams the product, or a team? Suggested revision: "You must connect to an instance of a team or a chat to make this request."
```suggestion
            "No Connected Resources": "You must connect to an instance of a team or a chat to make this request."
```

File: tabs/src/translations/en-US.json [hunk 6]
```diff
@@ -1,6 +1,58 @@
...
+            "ChatSettings.Read.Chat": "Get this chat's settings.",
+            "ChatSettings.ReadWrite.Chat": "Update this chat's settings.",
+            "ChatMessage.Read.Chat": "Get this chat's messages.",
+            "ChatMember.Read.Chat": "Get this chat's members.",
+            "Chat.Manage.Chat": "Manage this chat.",
+            "TeamsTab.Read.Chat": "Get this chat's tabs.",
+            "TeamsTab.Create.Chat": "Create tabs in this chat.",
+            "TeamsTab.Delete.Chat": "Delete this chat's tabs.",
+            "TeamsTab.ReadWrite.Chat": "Manage this chat's tabs.",
+            "TeamsAppInstallation.Read.Chat": "Get which apps are installed in this chat.",
```
Comment (Lauragra) [inline tabs/src/translations/en-US.json]: Revise as: "Get a list of the apps that are installed in this chat."
```suggestion
            "TeamsAppInstallation.Read.Chat": "Get a list of the apps that are installed in this chat.",
```

File: tabs/src/translations/zh-CN.json [hunk 7]
```diff
@@ -1,15 +1,70 @@
...
+            "Value": "(zh-cn) Value",
+            "Run query": "(zh-cn) RUN QUERY",
+            "Request body": "(zh-cn) Request Body",
+            "Request headers": "(zh-cn) Request Headers",
+            "Response body": "(zh-cn) Response Body",
+            "Response headers": "(zh-cn) Response Headers",
+            "Response": "(zh-cn) Response",
+            "Warning": "(zh-cn) To use Graph requests, you must connect the app to a resource in order to use resource-specific consent queries. Learn more here.",
```
Comment (Lauragra) [inline tabs/src/translations/zh-CN.json]: Same suggestion here: "You must connect the app to a resource in order to make Microsoft Graph requests that use resource-specific consent. For details, see {link to name of article}."
```suggestion
            "Warning": "(zh-cn) ou must connect the app to a resource in order to make Microsoft Graph requests that use resource-specific consent. For details, see {link to name of article (placeholder)}.",
```

File: tabs/src/translations/zh-CN.json [hunk 8]
```diff
@@ -1,15 +1,70 @@
...
+            "Add": "(zh-cn) ADD"
+        },
+
+        "Connected Resources": {
+            "Teams Channel ID": "(zh-cn) Teams Channel ID",
+            "Chat": "(zh-cn) Chat",
+            "Chat ID": "(zh-cn) Chat ID",
+            "No Connected Resources": "(zh-cn) No connected resources. Please connect to a Teams or Chats instance."
```
Comment (Lauragra) [inline tabs/src/translations/zh-CN.json]: Suggested: "You must connect to an instance of a team or a chat to make this request."
```suggestion
            "No Connected Resources": "(zh-cn) You must connect to an instance of a team or a chat to make this request."
```

File: tabs/src/translations/zh-CN.json [hunk 9]
```diff
@@ -1,15 +1,70 @@
...
+            "ChatSettings.Read.Chat": "(zh-cn) Get this chat's settings.",
+            "ChatSettings.ReadWrite.Chat": "(zh-cn) Update this chat's settings.",
+            "ChatMessage.Read.Chat": "(zh-cn) Get this chat's messages.",
+            "ChatMember.Read.Chat": "(zh-cn) Get this chat's members.",
+            "Chat.Manage.Chat": "(zh-cn) Manage this chat.",
+            "TeamsTab.Read.Chat": "(zh-cn) Get this chat's tabs.",
+            "TeamsTab.Create.Chat": "(zh-cn) Create tabs in this chat.",
+            "TeamsTab.Delete.Chat": "(zh-cn) Delete this chat's tabs.",
+            "TeamsTab.ReadWrite.Chat": "(zh-cn) Manage this chat's tabs.",
+            "TeamsAppInstallation.Read.Chat": "(zh-cn) Get which apps are installed in this chat.",
```
Comment (Lauragra) [inline tabs/src/translations/zh-CN.json]: Revise as: "Get a list of the apps that are installed in this chat."
```suggestion
            "TeamsAppInstallation.Read.Chat": "(zh-cn) Get a list of the apps that are installed in this chat.",
```

**Review by omersayshi — APPROVED** [review]

**Pull request merged**